datastore/concepts.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var gcloud = require('gcloud')({
+  projectId: process.env.TEST_PROJECT_ID
+});
+
+var datastore = gcloud.datastore();
+
+// [START Entities:incomplete_key]
```

Comment:
The snippets have to match exactly, so this should be // [START incomplete_key].